packages/x-charts/src/ChartsLabel/ChartsLabel.tsx

```diff
@@ -0,0 +1,74 @@
+'use client';
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { styled, SxProps, Theme } from '@mui/material/styles';
+import clsx from 'clsx';
+import { ChartsLabelClasses, useUtilityClasses } from './labelClasses';
+import { consumeThemeProps } from '../internals/consumeThemeProps';
+
+export interface ChartsLabelProps {
+  /**
+   * Style applied to legend labels.
+   * @default theme.typography.caption
+   */
+  labelStyle?: React.CSSProperties;
+  /**
+   * Override or extend the styles applied to the component.
+   */
+  classes?: Partial<ChartsLabelClasses>;
+  children?: React.ReactNode;
+  className?: string;
+  sx?: SxProps<Theme>;
+}
+
+const Root = styled('div', {
+  name: 'MuiChartsLabel',
+  slot: 'Root',
+  overridesResolver: (props, styles) => styles.root,
+})<{ ownerState: ChartsLabelProps }>(({ theme, ownerState }) => ({
+  ...theme.typography.caption,
+  color: (theme.vars || theme).palette.text.primary,
```

Comment:
We currently use theme colors for this, but the designs suggest other colors. I'm not sure how to proceed. If we follow the design, where do these colors come from?

https://www.figma.com/design/upttiqJUQYGUu3xo6w7COx/Design-refinement?node-id=45-334&node-type=frame&t=kteCWeOvyprNETmc-0